cllc-public-app/ClientApp/src/app/shared/components/org-structure/org-structure.component.ts

```diff
@@ -74,54 +78,64 @@ export class OrgStructureComponent implements OnInit {
   asLicenseeChangeLog(val): LicenseeChangeLog { return val; }
 
   updateNumberOfFiles(numberOfFiles: number, docType: string) {
-    if (!this.node.fileUploads) {
-      this.node.fileUploads = {};
+    if (!this.fileUploads) {
+      this.fileUploads = {};
     }
-    this.node.fileUploads[docType] = numberOfFiles;
+    this.fileUploads[docType] = numberOfFiles;
   }
 
   updateChildren(children: LicenseeChangeLog[], changeType: string) {
 
-    children = children || [];
-    this.node.children = this.node.children || [];
-    if (changeType === 'Leadership') {
-      this.node.children = [...children,
-      ...this.node.individualShareholderChildren,
-      ...this.node.businessShareholderChildren];
-    } else if (changeType === 'IndividualShareholder') {
-      this.node.children = [...children,
-      ...this.node.keyPersonnelChildren,
-      ...this.node.businessShareholderChildren
-      ];
-    } else if (changeType === 'BusinessShareholder') {
-      this.node.children = [...children,
-      ...this.node.individualShareholderChildren,
-      ...this.node.keyPersonnelChildren
-      ];
-    }
+    // children = children || [];
+    // this.node.children = this.node.children || [];
+    // if (changeType === 'Leadership') {
+    //   this.node.children = [...children,
+    //   ...this.node.individualShareholderChildren,
+    //   ...this.node.businessShareholderChildren];
```

Comment:
this function not used anymore?

Reply:
Yes. I will clean it out of  the code in the next PR.